.github/tests.yaml

```diff
@@ -0,0 +1,12 @@
+broker[docker]: "tests/foreman/ -v -k 'test_host_registration_end_to_end or test_positive_erratum_applicability or test_positive_upload_content'"
```

Comment:
Should we really need a verbose output?

Reply:
can be removed, not that it is necessary.